x-pack/plugins/siem/server/graphql/ecs/schema.gql.ts

```diff
@@ -93,47 +93,47 @@ export const ecsSchema = gql`
   }
 
   type Thread {
-    id: ToStringArray
+    id: ToNumberArray
     start: ToStringArray
```

Comment:
I couldn't seem to find the ECS docs for `thread.start`, but [process.start](https://www.elastic.co/guide/en/ecs/current/ecs-process.html) is a `Date`, so should this be a `ToDateArray`?